inference/wasm/tests/test-cases/shared.mjs

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { it, expect } from "vitest";
+import { TranslationsEngine } from "./engine/translations-engine.mjs";
+
+/**
+ * Runs a translation test, constructing a Translations Engine for the given
+ * sourceLanguage and targetLanguage, then asserting that the translation of
+ * the sourceText matches the expectedText.
+ *
+ * @param {Object} params - The parameters for the test.
```

Comment:
Thought: We should add a `tsconfig.json` file at some point for all this JS code. e.g. https://github.com/gregtatum/taskcluster-tools/blob/main/tsconfig.json

Reply:
Filed an issue: https://github.com/mozilla/translations/issues/930